build/version.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <IISAdministrationVersion>2.2.1</IISAdministrationVersion>
+    <IISAdministrationVersion>3.0.0</IISAdministrationVersion>
```

Comment:
@shirhatti the thinking here was that upgrading to a .NET Core 2.1 dependency was a big enough change to warrant the major version update. Thoughts ?

Reply:
talked to @bariscaglar and @shirhatti , decided on 2.3.0